packages/reporter/src/errors/errors.scss

```diff
@@ -59,6 +59,14 @@ $code-border-radius: 4px;
     text-align: center;
   }
 
+  .studio-err-wrapper {
```

Comment:
Can we only render `.studio-err-wrapper` when an error occurs vs always?

Reply:
Yeah this was the Cypress 9 implementation, not really ideal... I'll see if I can quickly fix this.

Reply:
Updated this to only render when the test errors